### ISSUE TITLE
Dead FAQ link

### ISSUE THREAD

**Issue by Avasam** [microsoft/MAPIStubLibrary#26]
The FAQ link https://mapistublibrary.codeplex.com/wikipage?title=FAQ is a dead link

It looks like the FAQ/wiki wasn't transfered over from CodePlex. Should the link be updated to link to newer Microsoft docs instead ? (like https://github.com/microsoft/MAPIStubLibrary/pull/27)

**Comment by stephenegriffin** [comment]
Yes - please submit any PRs you want and I'll approve if they look good. Thanks for contributing!

**Comment by Avasam** [comment]
Thing is idk what would be an equivalent new location ^^" If there even is one, those old FAQ might be lost forever :/

**Comment by telenieko** [comment]
All Hail the Wayback Machine!
it has a copy of the wiki, from 2017:

https://web.archive.org/web/20171226201430/https://mapistublibrary.codeplex.com/wikipage?title=FAQ
